worker/appm/store/store.go

```diff
@@ -1008,23 +1008,30 @@ func (a *appRuntimeStore) GetAppStatus(appID string) (pb.AppStatus_Status, error
 
 	appStatus := pb.AppStatus_RUNNING
 	serviceStatuses := a.GetAppServicesStatus(serviceIDs)
+	var statuses []string
```

Comment:
if all component status is `undeploy`, The application will be judged to be AppStatus_RUNNING.

Reply:
Suggest to improve the test case, here I feel the logic is problematic.

Reply:
ok